File: C AND C++/calculator.java
import java.util.Scanner;

public class Calculator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num1, num2, choice;

        System.out.print("Enter 1st Number: ");
        num1 = scanner.nextInt();
        System.out.print("Enter 2nd Number: ");
        num2 = scanner.nextInt();

        do {
            System.out.println("\n====MENU====");
            System.out.println("1. ADDITION");
            System.out.println("2. SUBTRACTION");
            System.out.println("3. MULTIPLICATION");
            System.out.println("4. DIVISION");
            System.out.println("5. MODULUS");
            System.out.println("6. EXIT");

            System.out.print("Enter Your Choice: ");
            choice = scanner.nextInt();

            switch (choice) {
                case 1:
                    System.out.println("Result: " + add(num1, num2));
                    break;
                case 2:
                    System.out.println("Result: " + subtract(num1, num2));
                    break;
                case 3:
                    System.out.println("Result: " + multiply(num1, num2));
                    break;
                case 4:
                    if (num2 != 0) {
                        System.out.println("Result: " + divide(num1, num2));
                    } else {
                        System.out.println("Division by zero is not allowed.");
                    }
                    break;
                case 5:
                    if (num2 != 0) {
                        System.out.println("Result: " + modulo(num1, num2));
                    } else {
                        System.out.println("Modulus by zero is not allowed.");
                    }
                    break;
                case 6:
                    break;
                default:
                    System.out.println("Invalid choice. Please try again.");
                    break;
            }

        } while (choice != 6);
    }

    public static int add(int a, int b) {
        return a + b;
    }

    public static int subtract(int a, int b) {
        return a - b;
    }

    public static int multiply(int a, int b) {
        return a * b;
    }

    public static int divide(int a, int b) {
        return a / b;
    }

    public static int modulo(int a, int b) {
        return a % b;
    }
}

File: C AND C++/calculator.cpp
#include<iostream>
using namespace std;

int add(int a, int b)
{
    int answer=a+b;
    cout<<answer;
    return 0;
}

int subtract(int a, int b)
{
    int answer=a-b;
    cout<<answer;
    return 0;
}
int multiply(int a, int b)
{
    int answer=a*b;
    cout<<answer;
    return 0;
}
int devide(int a, int b)
{
    int answer=a/b;
    cout<<answer;
    return 0;
}
int modulo(int a, int b)
{
    int answer=a%b;
    cout<<answer;
    return 0;
}
int main()
{
    int num1,num2,choice;
    cout<<"Enter 1st Number:";
    cin>>num1;
    cout<<"\nEnter 2nd Number:";
    cin>>num2;

do{
    cout<<"\n====MENUE===="<<endl;
    cout<<"1. ADDITION"<<endl;
    cout<<"2. SUBTACTION"<<endl;
    cout<<"3. MULTIPLICATION"<<endl;
    cout<<"4. DIVISION"<<endl;
    cout<<"5. MODULUS"<<endl;
    cout<<"6. EXIT"<<endl;

    cout<<"Enter Your Choice:";
    cin>>choice;

    switch(choice)
    {
        case 1: add(num1,num2); 
                break;
        case 2: subtract(num1,num2); 
                break;
        case 3: multiply(num1,num2); 
                break;
        case 4: devide(num1,num2); 
                break;
        case 5: modulo(num1,num2); 
                break;
        case 6: break;
        default: add(num1,num2); 
                 break;
    }

}while(choice!=6);
    


    return 0;
}